alarm/pom.xml

```diff
@@ -47,7 +41,12 @@
         <dependency>
             <groupId>com.gluonhq</groupId>
             <artifactId>charm-cloudlink-client</artifactId>
-            <version>${charm.version}</version>
+            <version>${charm.cloudlink.version}</version>
```

Comment:
We need to either use `charm.cloudlink.version` in other projects and introduce this property. Or, we can replace this with `charm.version` as used in other projects.

Reply:
I did include that property for all projects:

https://github.com/gluonhq/gluon-samples/pull/127/files?authenticity_token=VF2TpgKgxgArRghGi45kkbdXFln7cc4m8DsxwD%2FL77t9gjgt3ckoUIMCeYncNBGpWWPvf5SaPOwaKJ91G0k13g%3D%3D&file-filters%5B%5D=.xml&hide-deleted-files=true#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R16